aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -114,6 +116,16 @@ module AptosFramework::Token {
         maximum: Option<u64>,
     }
 
+    /// The royalty of a token
+    struct Royalty has copy, drop, store {
+        // The basis point and denominator is ROYAL_BASIS_POINT_DENOMINATOR (1,000,000)
+        royalty_basis_points: u64,
```

Comment:
I wonder if the name "basis point" is misleading. Basis point in finance is 1/100 of a percent. So the denominator is 10,000. But here we're using one million as the denominator... https://www.investopedia.com/terms/b/basispoint.asp

Reply:
I agree with @hariria. Or we should explicitly define what we mean by this, clearly spell out our own definition.

Reply:
unfortunately, we don't have float in move. 1% might be still too large for some high value token. use royalty_points now. 
I guess using 1 million as denominator might not be common. we should document this well.

Reply:
@areshand I guess what I'm saying is if we use the word "basis point" we should make the denominator 10,000. Otherwise, we should find another word to describe what we mean

Reply:
Like maybe micro_point? Micro = 10^-6. I dont really like the sound of that lol

Reply:
based on the your suggestions, I update the name to `royalty_points_per_million` to explicitly call out the denominator is 1million